spring-cloud-aws-messaging/src/main/java/org/springframework/cloud/aws/messaging/listener/SqsMessageDeletionPolicy.java

```diff
@@ -72,6 +73,12 @@
 	 * Deletes message when successfully executed by the listener method. If an exception
 	 * is thrown by the listener method, the message will not be deleted.
 	 */
-	ON_SUCCESS
+	ON_SUCCESS,
+
+	/**
+	 * If this is set in SqsListener, it will use global value set for specific QueueMessageHandlerFactory.
+	 * Default, if not changed is set to NO_REDRIVE.
+	 */
+	GLOBAL
```

Comment:
I think `DEFAULT` would be a better name. `global` -> `default` should be applied then in every other changed file.

Reply:
True default is better. Done